spec/unit/indexer/message_processor_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "spec_helper"
+
+RSpec.describe Indexer::MessageProcessor do
+  let(:content_item) { generate_random_example }
+  let(:message) { GovukMessageQueueConsumer::MockMessage.new(content_item) }
+  let(:logger) { Logging.logger[described_class] }
+
+  context "when successful" do
+    before do
+      expect(Indexer::ChangeNotificationProcessor)
+        .to receive(:trigger)
+        .with(message.payload)
+        .and_return(:accepted)
+    end
+
+    it "logs some messages" do
+      expected_messages = [
+        /Processing message \[\]: {"content_id":"#{content_item['content_id']}"/,
+        /Finished processing message/,
+      ]
+
+      expected_messages.each do |message|
+        expect(logger).to receive(:info).with(message)
+      end
+
+      described_class.new.process(message)
+    end
+
+    it "increments the statsd counter" do
+      expect(Services.statsd_client).to receive(:increment).with("message_queue.indexer.accepted")
+      described_class.new.process(message)
+    end
```

Comment:
As I understand, this is actually a no-op now as we're not running statsd anymore. Though I think we may be best to just leave this in as it'd feel inconsistent to chop bits out and there's quite a lot of statsd in the codebase.